workflow/README.md

```diff
@@ -5,14 +5,14 @@ Automate your data processing tasks using Peak Workflows.
 We provide Python and R versions of the images:
 - [Python](./python)
   - [standard-pack](./python/standard-pack)
-    - [workflow-python-standard-pack-v1](./python/standard-pack/v1)
+    - [workflow-python-standard-pack-v1](./python/standard-pack/1.0.0-base-python-3.8.13)
```

Comment:
Hey, these names would need update and we will have to add all the new images as well